tests/integration/go/yaml/manifests/crontab.yaml

```diff
@@ -0,0 +1,36 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
How are we automating testing this if the CRD needs to be installed?

Should we install this (or a simpler) CRD as part of the Go program?

Reply:
The test will install both the CRD and CR from this manifest. The built in retry allows it to succeed even though they’re in the same file.

Reply:
I can't tell where this file is actually getting used - are we running a test that uses this?

Reply:
Yeah, it gets applied by the `ConfigGroup` in the yaml test.